remoulade/api/main.py

```diff
@@ -55,3 +67,8 @@ def remoulade_exception(e):
 @app.errorhandler(HTTPException)
 def http_exception(e):
     return {"error": str(e)}, e.code
+
+
+@app.errorhandler(ValidationError)
+def validation_error(e):
+    return {"error": str(e)}, 400
```

Comment:
you should use `e.normalized_messages()` instead and str(e) (better to use json than a string). this way you donnot need the wierd eval in the tests.